### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by TomPed**
We want a more standard style for the Codecov example `READMD.md`s. I've added and moved around a few things. @nixel2007 it looks like you've done some work on this. Would you mind reviewing the changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,59 @@
...
-[4]: https://github.com/codecov/codecov-bash
+# [Codecov][1] VSCode Extension Example
 
 [![codecov](https://codecov.io/gh/codecov/example-typescript-vscode-extension/branch/master/graph/badge.svg)](https://codecov.io/gh/codecov/example-typescript-vscode-extension)
 [![Build Status](https://travis-ci.org/codecov/example-typescript-vscode-extension.svg?branch=master)](https://travis-ci.org/codecov/example-typescript-vscode-extension)
 
-Run npm install:
-```shell
-npm install
-npm run vscode:prepublish
```
Comment (nixel2007) [inline README.md]: These steps above is required to make testing posible. if `npm install` is obvious (and AFAIK travis run this step automaticaly), the `npm run vscode:prepublish` is required to compile ts-files to js.

File: README.md [hunk 2]
```diff
@@ -1,28 +1,59 @@
...
+    "enabled": true,
+    "relativeSourcePath": "../src",
+    "relativeCoverageDir": "../../coverage",
+    "ignorePatterns": ["**/node_modules/**"],
+    "includePid": false,
+    "reports": ["json", "html", "lcov"],
+    "verbose": false
+}
 ```
+Run `node ./node_modules/vscode/bin/test` (make sure `vscode` is installed via `npm`)
```
Comment (nixel2007) [inline README.md]: package.json has test-task, so you can simply run `npm test`

Comment (TomPed) [reply README.md]: I was thinking of giving base commands incase they don't want to use npm scripts. Do you think that if I have both that would be ok?

Comment (TomPed) [reply README.md]: I can add the `npm` call back.

Comment (nixel2007) [reply README.md]: sure

**Comment by TomPed** [comment]
Updated.

**Comment by nixel2007** [comment]
One more (annoying) tip. These lines [link](https://github.com/codecov/example-typescript-vscode-extension/blob/master/.travis.yml#L9-L13) is mandatory to make VSCode work on travis :(

**Comment by TomPed** [comment]
Ew, that is gross. No worries. I'll add that!

**Comment by nixel2007** [comment]
So... I think it'll be a good adwise to clone this repo entirely and update only ts-code :D

**Comment by TomPed** [comment]
Updated.

**Comment by nixel2007** [comment]
LGTM!

**Pull request merged**